documents/policies/external/Travel Reimbursement.md

```diff
@@ -29,7 +29,7 @@ The purpose of this policy is to define the conditions under which the travel ex
    3. Alaska, Hawaii, and US territories – [US Department of Defense OCONUS Per Diem Rates](https://www.travel.dod.mil/Travel-Transportation-Rates/Per-Diem/Per-Diem-Rate-Lookup/)
 2. Lodging expenses for travel to a competition can only be claimed for the night before a competition begins, nights during a competition, and the night after a competition ends. Lodging expenses for travel on official WCA business outside of a competition can only be claimed for nights before or after days where WCA business is undertaken.
 3. Meal expenses can only be claimed for days during a competition, days during which official WCA business is undertaken, or days on which WCA-funded travel is taking place.
-4. If anyone not explicitly named and authorized for WCA-funded travel is charged for a meal on the same receipt as someone whose travel is being funded, the meal expense must be prorated to only cover the portion of the food for the funded staff member(s) and a prorated portion of the tip (if applicable).
+4. If anyone not explicitly named and authorized for WCA-funded travel is charged for a meal on the same receipt as someone whose travel is being funded, the meal expense must be prorated to only cover the portion of the food for the funded WCA Volunteer(s) and a prorated portion of the tip (if applicable).
```

Comment:
```suggestion
4. If anyone not explicitly named and authorized for WCA-funded travel is charged for a meal on the same receipt as someone whose travel is being funded, the meal expense must be prorated to only cover the portion of the food for the funded individual(s) and a prorated portion of the tip (if applicable).
```
Since non-WCA Volunteers can be funded individuals (1.3 allows Trainees to be funded). 
@EdHollingdale

Reply:
It still baffles me why we aren't defining trainee delegates as WCA Volunteers too. But in that case your suggestion Hex is sensible (it fixes an inconsistency in the previous policy).